### PR TITLE
Course Post Acceptance Test(ShouldPostCourseAsync)

### PR DESCRIPTION
**Pull request by tajwal**
Course Post Acceptance Test(ShouldPostCourseAsync)

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.Logic.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+            Course actualCourse =
+                await this.courseBroker.GetCourseByIdAsync(inputCourse.Id);
+
+            // then
+            actualCourse.Should().BeEquivalentTo(expectedCourse);
+
+            await this.courseBroker.DeleteCourseByIdAsync(actualCourse.Id);
+        }
+
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.Logic.cs:35]: Remove empty new lines

File: OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.cs:6]: Remove extra empty lines

File: OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                .OnProperty(course => course.CreatedBy).Use(posterId)
+                .OnProperty(course => course.UpdatedBy).Use(posterId)
+                .OnProperty(course => course.CreatedDate).Use(now)
+                .OnProperty(course => course.UpdatedDate).Use(now)
+                .OnType<DateTimeOffset>().Use(GetRandomDateTime());
+
+            return filler;
+        }
+    }
+
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Acceptance/APIs/Courses/CoursesApiTests.cs:48]: Remove extra empty lines

File: .gitignore [hunk 4]
```diff
@@ -337,4 +337,14 @@ ASALocalRun/
 .localhistory/
 
 # BeatPulse healthcheck temp database
-healthchecksdb
\ No newline at end of file
+healthchecksdb
+/.vscode/launch.json
+/.vscode/tasks.json
+/OtripleS.Web.Api/Migrations/20200719090247_AddCourses.cs
+/OtripleS.Web.Api/Migrations/20200719090247_AddCourses.Designer.cs
```
Comment (hassanhabib) [inline .gitignore:344]: Why do we want to ignore our migration files?

Comment (tajwal) [reply .gitignore:344]: @hassanhabib I had these files in my project and showing some changes that why i have ignored, and I wanted to submit my won changes only.

Comment (hassanhabib) [reply .gitignore:344]: @tajwal you need to pull a fresh copy from master from this repo then apply your changes.

**Comment by hassanhabib** [comment]
Shutting this PR down for being overdue - replaced with https://github.com/hassanhabib/OtripleS/pull/61

**Pull request closed without merging**